homework/homework_14/homework_14_1.py

```diff
@@ -0,0 +1,34 @@
+"""Homework 14"""
+# Task # 1
+
+try:
+    with open("students.txt", "w+", encoding="utf-8") as students_file:
+        content = students_file.write("student1 group1 5\n"
+                                      "student2 group1 5\n"
+                                      "student3 group1 5\n"
+                                      "student4 group2 5\n"
+                                      "student5 group2 4\n"
+                                      "student6 group2 3\n")
```

Comment:
Store information separately:
```suggestion
    students_data = [
        "Serena Williams","1A",[5,4,4,3],
        "Lionel Messi","2B",[4,3,5,2],
        "Kobe Bryant","1A",[3,5,4,1]
    ]
    ...
    with open("students.txt", "w+", encoding="utf-8") as students_file:
        students_file.write('\n'.join(students_data) + '\n')
```